analyzers/tests/SonarAnalyzer.TestFramework/Build/ProjectBuilder.cs

```diff
@@ -122,6 +126,9 @@ private string GetTestCaseFileRelativePath(string path)
 
     private static ProjectBuilder AddAdditionalDocument(Project project, string fileName, string fileContent) =>
         FromProject(project.AddAdditionalDocument(fileName, fileContent).Project);
+
+    public ProjectBuilder AddAnalyzerConfigDocument(string editorConfigPath, SourceText text) =>
+        FromProject(project.AddAnalyzerConfigDocument(editorConfigPath, text, filePath: editorConfigPath).Project);
```

Comment:
Nitpick: I would take a `string `instead of `SourceText`.
```suggestion
    public ProjectBuilder AddAnalyzerConfigDocument(string editorConfigPath, string content) =>
        FromProject(project.AddAnalyzerConfigDocument(editorConfigPath, SourceText.From(content), filePath: editorConfigPath).Project);
```
It is fewer dependencies in the Verifier and I think this is closer to what the other methods do.